res/container-tests/Containerfile.ubi10

```diff
@@ -0,0 +1,35 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+VOLUME /payload
+
+RUN yum update -y && \
+    yum install python3 python39 python3-virtualenv make -y && \
```

Comment:
python312?

Reply:
yes, I've just created files without changes. keeping that for anyone who gets to it first :)

Reply:
problem is that ubi10 image will not be available till rhel 10 GA I guess.

Reply:
Not even for beta?

Reply:
no idea honestly.

Reply:
It definitely seems like it isn't available, even for beta.